grails-app/conf/UrlMappings.groovy

```diff
@@ -62,6 +62,8 @@ class UrlMappings {
 
 		"/api/v1/modules"(resources: "moduleApi")
```

Comment:
On this line, `resources: "moduleApi"` auto-maps the whole CRUD set of endpoints. We don't want that, so we could be more explicit about what is available.

```suggestion
		"/api/v1/modules"(controller: "moduleApi", "action": "index")
```

Reply:
Does it make a difference since those methods haven't been defined anyways?